frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -316,23 +342,20 @@ const ManageSchedulePage = ({
     ) => {
       if (editQuery) {
         const updatedAttributes = deepDifference(formData, editQuery);
-        dispatch(
-          selectedTeamId
-            ? teamScheduledQueryActions.update(editQuery, updatedAttributes)
-            : globalScheduledQueryActions.update(editQuery, updatedAttributes)
-        )
+
+        const edit = selectedTeamId
```

Comment:
May want to consider a more descriptive name for this variable like `editResponse`